### PR TITLE
Deprecate version 9.6 which is EOL

### PR DESCRIPTION
**Pull request by phracek**
This commit deprecates version PostgreSQL 9.6 which is EOL.

Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
The test passes on new Jenkins but Git Hub Status was not updated.

Let's try one more time.

I have updated Jenkin's job.

[test]

File: README.md [hunk 1]
```diff
@@ -77,20 +76,12 @@ In this repository [distgen](https://github.com/devexp-db/distgen/) is used for
...
-see [usage documentation](https://github.com/sclorg/postgresql-container/tree/generated/9.6).
-
 For information about usage of Dockerfile for PostgreSQL 10,
 see [usage documentation](https://github.com/sclorg/postgresql-container/tree/generated/10).
 
 For information about usage of Dockerfile for PostgreSQL 12,
 see [usage documentation](https://github.com/sclorg/postgresql-container/tree/generated/12).
 
-For versions which are not supported anymore:
-[PostgreSQL 9.2](https://github.com/sclorg/postgresql-container/blob/f213e5d0/9.2)
```
Comment (hhorak) [inline README.md:90]: Is this removal deliberate? It actually makes sense to keep such references for anybody who needs some EOL version and does not care it is deprecated already.

**Comment by phracek** [comment]
Deprecated version, 9.6, is added into not supported versions. Let's get merge it.
